docs/cloud-linter/getting-started.mdx

```diff
@@ -0,0 +1,32 @@
+---
+sidebar_position: 2
+sidebar_label: Getting Started
+title: Getting Started
+pagination_next: null
+description: Explore Cloud Linter and see what insights it can provide you about your cloud environment.
+hide_title: true
+---
+
+# Getting Started
+
+If you haven't already, install the momento cli. Instructions for installation and usage are available in [this repo](https://github.com/momentohq/momento-cli).
+
+Once the cli is installed, use the `cloud-linter` command to run the linter against your AWS environment. Make sure to use the AWS_PROFILE environment variable to specify the AWS account you want to use.
+
+<div>
+    <video width="100%" height="100%" playsInline autoPlay muted loop>
+        <source src="/linter_preview.mp4" type="video/mp4" />
+    </video>
+</div>
```

Comment:
maybe they are in the video but you don't list the actual command here. I think you should either list it, or link them to the console at the very beginning and ask them to follow the instructions from the console rather than repeating the instructions here.